kubernetes/omsagent.yaml

```diff
@@ -746,14 +746,23 @@ spec:
       port: 25227
       targetPort: in-rs-tcp
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can you please add comment about how to handle this version across k8s versions and also helpful when we make changes to AKS yaml and using this yaml for local development?

Reply:
> can you please add comment about how to handle this version across k8s versions ?. This will be helpful when we make changes to AKS yaml and using this yaml for local development.